pipelines/controller/pom.xml

```diff
@@ -31,6 +31,15 @@
   <properties>
     <java.version>17</java.version>
     <spring-boot.version>3.1.3</spring-boot.version>
+    <sonar.java.coveragePlugin>jacoco</sonar.java.coveragePlugin>
+    <sonar.dynamicAnalysis>reuseReports</sonar.dynamicAnalysis>
+    <sonar.jacoco.reportPath>${project.basedir}/target/jacoco.exec</sonar.jacoco.reportPath>
```

Comment:
The property `sonar.jacoco.reportPath` seems to be deprecated, can we use the `sonar.coverage.jacoco.xmlReportPaths` path instead? Details can be found [here](https://community.sonarsource.com/t/coverage-test-data-importing-jacoco-coverage-report-in-xml-format/12151)